.github/workflows/canary.yml

```diff
@@ -0,0 +1,41 @@
+name: Canary Testing
+on:
+  schedule:
+    - cron: '0 * * * *'
+  workflow_dispatch: {}
+
+jobs:
+  install:
+    name: Installation User Flow
+    strategy:
+      fail-fast: true
+      matrix:
+        runner:
+          - ubuntu-latest
+          - windows-latest
+          - macos-latest
+        node:
+          - 18
+          - 19
+        install_command:
+          - npm install -g winglang
```

Comment:
Was initially going to add yarn here as well, but found out that yarn got rid of global installs at some point.
I think maybe this is initially good enough, since `npm install -g winglang` is what we actually tell people to do.